<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AllFlightsPage</class>
 <widget class="QWidget" name="AllFlightsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>472</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>40</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* ==================== 
   1. 顶部筛选栏样式 
   ==================== */
QFrame#topBar {
    background-color: white;
    border-bottom: 1px solid #E0E0E0; /* 底部淡灰色分割线 */
}

/* 输入框样式 */
QLineEdit {
    border: 1px solid #DCDFE6;
    border-radius: 4px;
    padding: 0 10px; /* 文字内边距 */
    font-size: 14px;
    color: #333;
    background-color: white;
}

QLineEdit:focus {
    border: 1px solid #0078D7; /* 选中时变蓝 */
}

/* 箭头 Label */
QLabel {
    color: #909399;
    font-weight: bold;
}

/* 查询按钮 */
QPushButton#btnSearch {
    background-color: #0078D7; /* 携程蓝 */
    color: white;
    font-size: 15px;
    font-weight: bold;
    border-radius: 4px;
    border: none;
}

QPushButton#btnSearch:hover {
    background-color: #005A9E; /* 悬停深蓝 */
}

QPushButton#btnSearch:pressed {
    background-color: #004C85;
}

/* ==================== 
   2. 滚动区域样式 
   ==================== */
QScrollArea {
    background-color: #F5F7FA; /* 整体背景淡灰 */
    border: none;
}

/* 滚动条内部容器 */
QWidget#scrollAreaWidgetContents {
    background-color: #F5F7FA; /* 确保里面也是淡灰 */
}

/* 滚动条美化 (可选，去掉就是系统默认) */
QScrollBar:vertical {
    border: none;
    background: #F5F7FA;
    width: 8px;
    margin: 0px;
}
QScrollBar::handle:vertical {
    background: #C0C4CC;
    border-radius: 4px;
    min-height: 20px;
}
QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    height: 0px;
}</string>
  </property>
  <widget class="QFrame" name="topBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLineEdit" name="lineEditDep">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>出发城市</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditArr">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>到达城市</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSearch">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="DateSelector" name="dateSelector" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>⇄</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>571</width>
     <height>341</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>571</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DateSelector</class>
   <extends>QWidget</extends>
   <header location="global">DateSelector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
